scitasexternal/packages/ansys/package.py

```diff
@@ -69,10 +69,12 @@ def setup_environment(self, spack_env, run_env):
         run_env.prepend_path('PATH', join_path(ansys_prefix, 'autodyn/bin'))
         run_env.prepend_path('PATH', join_path(ansys_prefix, 'fluent/bin'))
         run_env.prepend_path('PATH', join_path(ansys_prefix, 'polyflow/bin'))
-        run_env.prepend_path('PATH', join_path(ansys_prefix, 'tgrid/bin'))
+        if '17.1' in ansys_prefix: run_env.prepend_path('PATH', join_path(ansys_prefix, 'tgrid/bin'))
         run_env.prepend_path('PATH', join_path(ansys_prefix, 'Framework/bin/Linux64'))  # noqa: E501
         run_env.prepend_path('PATH', join_path(ansys_prefix, 'icemcfd/linux64_amd/bin'))  # noqa: E501
         
         run_env.prepend_path('LD_LIBRARY_PATH', join_path(ansys_prefix, 'Framework/bin/Linux64'))  # noqa: E501
-        run_env.prepend_path('LD_LIBRARY_PATH', join_path(ansys_prefix, 'polyflow/polyflow17.1.0/lnamd64/libs'))  # noqa: E501
+        if '19.2' in ansys_prefix: run_env.prepend_path('LD_LIBRARY_PATH', join_path(ansys_prefix, 'polyflow/polyflow19.2.0/lnamd64/libs'))  # noqa: E501
```

Comment:
I think the following would work for all the current versions we have and probably future versions without change.
```suggestion
        run_env.prepend_path('LD_LIBRARY_PATH', join_path(ansys_prefix, 'polyflow/polyflow{0}.0/lnamd64/libs'.format(vers.up_to(2).dotted)))  # noqa: E501
```

Reply:
Done! Thanks a lot!